tests/class-wp-test-rest-controller-testcase.php

```diff
@@ -39,4 +39,8 @@ public function tearDown() {
 
 	abstract public function test_get_item_schema();
 
+	public function test_rest_url_for_leading_slash( $url, $path ) {
+		// Make sure path for rest_url has a leading slash for proper resolution.
+		$this->assertTrue( 0 === strpos( $path, '/' ) );
```

Comment:
Yeah, I thought of that not sure why I never put it in.  The last thing is that this function is called on every subclass as though it is a test so you get an exception thrown because no parameters are passed into it.  I tried to make it protected but then the subclasses can't access it.  Should I change this to self?